raven.lua

```diff
@@ -484,7 +485,7 @@ function _M.gen_capture_err(self)
 end
 
 -- HTTP request template
-local xsentryauth_http = "POST %s HTTP/1.0\r\nHost: %s\r\nConnection: close\r\nContent-Type: application/json\r\nContent-Length: %d\r\nUser-Agent: %s\r\nX-Sentry-Auth: Sentry sentry_version=5, sentry_client=%s, sentry_timestamp=%s, sentry_key=%s, sentry_secret=%s\r\n\r\n%s"
+local xsentryauth_http = "POST %s HTTP/1.0\r\nHost: %s\r\nConnection: close\r\nContent-Type: application/json\r\nContent-Length: %d\r\nUser-Agent: %s\r\nX-Sentry-Auth: Sentry sentry_version=7, sentry_client=%s, sentry_timestamp=%s, sentry_key=%s, sentry_secret=%s\r\n\r\n%s"
```

Comment:
To jump in here, there's not really a reason to bump the version here unless it's leveraging a feature of the newer protocol versions. This version doesn't correlate directly to the major version of Sentry server, and arbitrarily bumping it to `7` will break compatibility on older versions of the server for potentially no reason.

But, if you did bump it to `8`, it would 100% stop working since we don't have a protocol version 8 and all events would get rejected.